### ISSUE TITLE
Huge bumps in learning curves

### ISSUE THREAD

**Issue by tbraeckevelt** [mir-group/nequip#73]
Dear developers

I have a problem while training the MLP. I get large bumps for the MAE of the forces. I know that it is not unusual to get some bumps while training, but usually the error goes back to the original value before the bump, quite fast. This is does not happen, it takes a few hours and in certain cases a new bump occurs befor reaching the old optimal error. See for example the figure below.

![LearningCurve](https://user-images.githubusercontent.com/85891250/130612926-c26df473-14a0-4800-bcbc-83ed06cdd561.png)

I know that I could change for example learning rate, or batch size or even restart the training from the optimal value before the bump. But I was wondering if this is something you saw while training MLPs or if you know what might cause this?

I ask because I am using almost the default settings of the "full.yaml" you provided (I only changed r_max to 6.0), so I would think that the settings are already quite good. However, I got this strange behavior for two different systems (CsPbI3 and FAPbI3) and two different training sets sizes (300 and 15000 structures). In the zip.file you can find the full.yaml file and the logs of the training belonging to the figure above.

[in_and_output_data.zip](https://github.com/mir-group/nequip/files/7039157/in_and_output_data.zip)

Kind regards

Tom
